pallets/liquid-staking/src/lib.rs

```diff
@@ -434,6 +416,26 @@ pub mod pallet {
             Ok(().into())
         }
 
+        #[pallet::weight(0)]
+        #[transactional]
+        pub fn update_bonded_amount(
+            origin: OriginFor<T>,
+            #[pallet::compact] bonded_amount: BalanceOf<T>,
+        ) -> DispatchResultWithPostInfo {
+            T::RelayOrigin::ensure_origin(origin)?;
+            let exchange_rate = Rate::checked_from_rational(
+                bonded_amount,
+                T::Assets::total_issuance(Self::liquid_currency()?),
+            )
+            .ok_or(Error::<T>::InvalidExchangeRate)?;
+            let old_exchange_rate = Self::exchange_rate();
+            if exchange_rate > old_exchange_rate {
+                ExchangeRate::<T>::put(exchange_rate);
+                Self::deposit_event(Event::<T>::ExchangeRateUpdated(exchange_rate));
+            }
```

Comment:
1/3 
there may be a potential bug:
suppose when update_bonded_amount, in the meantime, someone unstake, so `T::Assets::total_issuance(Self::liquid_currency()?)` will decrease too.
so `new_exchange_rate` is still greater than old one.

Reply:
So do you think it's better to update exchange_ rate while doing `settlement`, I guess it makes more sense